namespaces/openstack.yml

```diff
@@ -0,0 +1,120 @@
+version: 2016.03.10.0
+
+defaults:
+  type: string
+  doc_values: true
+  index: not_analyzed
+
+namespace:
+  name: openstack
+  type: group
+  description: >
+    Mapping specifically for openstack. Inherits all the fields from
+    __default__.
+  fields:
+
+    - name: component_name
+      type: string
+      description: >
+        The name of the component that produced this log message.  For
+        example, "neutron.agent.linux.utils" or "nova.compute.api" or
+        "nova.policy".
+      norms:
+        enabled: True
+
+    # BEGIN oslo context
+    #
+    # Most openstack services log a standard context string that
+    # includes the follow fields.  See the defaults for
+    # logging_context_format_string and logging_user_identity_format
+    # in http://docs.openstack.org/developer/oslo.log/opts.html
+
+    - name: request_id
+      type: string
+      description: >
+        TODO.
+      norms:
+        enabled: True
+
+    - name: user_uuid
+      type: string
+      description: >
+        TODO.
+      norms:
+        enabled: True
+
+    - name: project_uuid
+      type: string
+      description: >
+        TODO.
+      norms:
+        enabled: True
+
+    - name: domain
```

Comment:
Is this the domain name or uuid?  Same question for user_domain and project_domain fields below.